lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
+    description: "enforces getter/setter pairs in objects",
+    ruleType: "Best Practices",
+    recommended: false,
+    default: [{getWithoutSet: false, setWithoutGet: true}]
```

Comment:
There's an official [`default`](http://json-schema.org/latest/json-schema-validation.html#anchor101) metadata property in the spec. Since it's freeform, we'd want to add a step in the tests to parse the supplied value and verify it against the schema.